db/migrate/20230105100537_change_type_of_lesson_rating.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeTypeOfLessonRating < ActiveRecord::Migration[7.0]
+  def change
+    change_column :lesson_ratings, :rating, :integer
```

Comment:
Why type is changed to integer?

Reply:
@fooooxmr
Because the user can't put a float type rating, e.g. 4.5